volume/samp_app/app/resources/questions/QuestionsController.jl

```diff
@@ -10,9 +10,13 @@ end
 
 function create()
   Question(question = @params(:question_precise), author = @params(:question_author)) |> save 
-  redirect_to(:questions) 
```

Comment:
why is it removed?
I think it is better to redirect to top when created question.

Reply:
resumed